.github/ISSUE_TEMPLATE/kubernetes_bump.md

```diff
@@ -86,3 +94,7 @@ run the Cluster API controllers on the new Kubernetes version.
   * Prior art: #7193
 * [ ] Bump conversion-gen via `CONVERSION_GEN_VER` in `Makefile`
   * Prior art: #7118
+
+After release of CAPI v1.X.x (that supports Kubernetes v1.Y):
+
+* [ ] Bump WorkloadKubernetesVersion for test When testing clusterctl upgrades using ClusterClass (v1.X=>current) [ClusterClass]
```

Comment:
I think this can go (we'll do it as part of prepare main branch instead already)